jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/group/GroupServiceRest.java

```diff
@@ -3,177 +3,268 @@
+                                         @ApiParam(value = "The resource template name") @PathParam("resourceTemplateName") final String resourceTemplateName,
+                                         @ApiParam(value = "boolean value for token replaced") @QueryParam("tokensReplaced") final boolean tokensReplaced);
 
     @PUT
     @Path("/{groupName}/jvms/resources/preview/{resourceTemplateName}")
     @Consumes(MediaType.TEXT_PLAIN)
-    Response previewGroupJvmResourceTemplate(@PathParam("groupName") String groupName,
-                                             @PathParam("resourceTemplateName") final String resourceTemplateName,
-                                             String template);
+    @ApiOperation(value = "Preview group jvm resource template",
+            response = String.class
+    )
+    @ApiResponses(@ApiResponse(code = 500, message = "Failed to preview group JVM resource template"))
+    Response previewGroupJvmResourceTemplate(@ApiParam(value = "The group name to query") @PathParam("groupName") String groupName,
+                                             @ApiParam(value = "The resource template name") @PathParam("resourceTemplateName") final String resourceTemplateName,
+                                             @ApiParam(value = "The group JVM resource template") String template);
 
     @PUT
     @Path("/{groupName}/jvms/resources/template/{resourceTemplateName}")
     @Consumes(MediaType.TEXT_PLAIN)
-    Response updateGroupJvmResourceTemplate(@PathParam("groupName") final String groupName,
-                                            @PathParam("resourceTemplateName") final String resourceTemplateName,
-                                            final String content);
+    @ApiOperation(value = "Update group JVM resource template",
+            response = String.class
+    )
+    @ApiResponses(@ApiResponse(code = 500, message = "Failed to update group JVM resource template"))
+    Response updateGroupJvmResourceTemplate(@ApiParam(value = "The group name to query") @PathParam("groupName") final String groupName,
+                                            @ApiParam(value = "The resource template name") @PathParam("resourceTemplateName") final String resourceTemplateName,
+                                            @ApiParam(value = "The updated group JVM resource template content") final String content);
 
 
     /****************************
      * ** Web Server Templates ***
      ****************************/
     @PUT
     @Path("/{groupName}/webservers/conf/{resourceFileName}")
-    Response generateAndDeployGroupWebServersFile(@PathParam("groupName") final String groupName,
-                                                  @PathParam("resourceFileName") @DefaultValue("httpd.conf")final String resourceFileName,
+    @ApiOperation(value = "Generate and deploy group webservers file",
```

Comment:
I believe the description should be more verbose, the information provided can be taken from the method name itself